README.md

```diff
@@ -228,6 +228,8 @@ yarn run test-cypress-console
 This will launch the Cypress Test Runner UI in the `console` package, where you can run one or all cypress tests.
 By default, it will look for Chrome in the system and use it, but if you want to use Firefox instead, set `BRIDGE_E2E_BROWSER_NAME` environment variable in your shell with the value `firefox`.
 
+Important:  when testing with authentication, set `BRIDGE_KUBEADMIN_PASSWORD` and `OAUTH_BASE_ADDRESS` environment variables in your shell.
```

Comment:
Good catch.  I initially had done so for CI when I thought `OAUTH_BASE_ADDRESS` differed there, too.  In trying to get CI working, I removed it entirely rather than moving it.